VK_App/VK_App/Controller/FriendsListTableViewController.swift

```diff
@@ -23,6 +25,16 @@ final class FriendsListTableViewController: UITableViewController {
         User(userName: "Антонио Бандерас", userImageName: "scroll6")
     ]
 
+    private var sections: [Character: [String]] = [:]
+    private var sectionsTitle: [Character] = .init()
```

Comment:
```suggestion
    private var sectionsTitle: [Character] = []
```
Только так = [], литер должен был отреагировать